README.md

```
@@ -1,2 +1,57 @@
 # mongoq
 Build MongoDB queries ⚙️
+
+
+## Example usage:
+```go
+package main
+
+import (
+	"context"
```

Markdownlint (0.37.0)

Column: 1
Hard tabs

(MD010, no-hard-tabs)